cmd/peek-submission/main.go

```diff
@@ -89,7 +89,7 @@ func main() {
 
 		fmt.Println(output)
 
-		util.Exit(2)
+		log.Exit(2)
```

Comment:
Leave this one as is.
Oliver already has changes lined up in #101, and we'll be nice and try to avoid conflicts for him.